### ISSUE TITLE
Plank-config fails

### ISSUE THREAD

**Issue by joaoherberto** [eericart/Plank-Config#7]
If anyone is still maintaining this app? Installing this application does nothing because the uses the command "plank-config" which does absoluetely nothing! The actual command is "plank --preferences".
Can someone commit this simple change?

**Comment by ghost** [comment]
Plank already has a configuration window. Just Hold control and right-click on the plank dock. There should be a menu that pops up. Then click Preferences.

That is probably why this app hasn't been maintained for a few years.

**Comment by joaoherberto** [comment]
Correct, but the average user will never figured that out. How hard can it be to just edit two characters?
